recipes/morphometrics/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "morphometrics" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/morphometrics-{{ version }}.tar.gz
+  sha256: 096ed657ef55aa1ff61e1fedfcde4ab882fcf115dbe488944da27194bc8e652a
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools-scm
+  run:
+    - cython
+    - hdbscan
+    - jinja2 >=2.10,<3.1
+    - leidenalg
+    - napari
```

Comment:
Will this automatically install a qt backend?

Reply:
by default if nothing is specific it will install pyqt as it has higher "precedence" but there are packages for pyside2

so

`conda install "napari -c conda-forge"` or `conda install "napari=*=pyqt* -c conda-forge"` -> pyqt
`conda install "napari=*=pyside2* -c conda-forge"` -> pyside

I think @jaimergp can provide more details

Reply:
Got it. Thanks!